content/en/error_tracking/guides/enable_infra.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Enable Infrastructure Monitoring
+further_reading:
+- link: "/infrastructure"
+  tag: "Documentation"
+  text: "Learn about Infrastructure Monitoring"
+- link: "/error_tracking/guides/enable_apm"
+  tag: "Guide"
+  text: "Enable APM"
+---
+
+This guide explains how to update the Datadog Agent configuration to enable [Infrastructure monitoring][1].
```

Comment:
Can you add 1-2 sentences about the benefits of enabling infrastructure monitoring with backend error tracking?